Dockerfile.template

```diff
@@ -0,0 +1,30 @@
+# install a Debian image with Python 3 installed
+FROM balenalib/%%BALENA_MACHINE_NAME%%-python:3-stretch-run
+
+RUN install_packages apt-utils
+RUN install_packages i2c-tools
+RUN install_packages python3-smbus
```

Comment:
@Bucknalla I would put these into one line since each `install_packages` does an apt update, it is way slower this way. Recommend doing:
`RUN install_packages apt-utils i2c-tools python3-smbus`